src/pages/dashboard/AccessFiltering.tsx

```diff
@@ -44,15 +48,19 @@ const AccessFiltering: FC<AccoessFilteringProps> = ({ queryObject, setQueryObjec
             name={'access'}
           />
         }
-        label={t('dashboard.access_filter')}
+        label={t('dashboard.access_filter_limited')}
         onChange={(event) => {
           changeSelected(event);
         }}
       />
       <HelperTextPopover
         ariaButtonLabel={t('explanation_text.access_filtering_label')}
         popoverId={'access-filtering-explainer'}>
-        <Typography>{`${t('explanation_text.access_filtering')} `}</Typography>
+        {/**
+         original label:
```

Comment:
når du skrive "orginal label" - er det fordi vi skal sette tilbake til det senere når backend er på plass ?  I tilfelle er det kanskje bedre å skrive det for å forstå hvorfor noe er kommentert ut

Reply:
Fikset